src/test/java/org/folio/settings/server/storage/SettingsStorageTest.java

```diff
@@ -29,17 +35,17 @@ public void getLimitsFromGlobal() {
     JsonArray perms = new JsonArray()
         .add("a")
         .add("settings.global.write.s1")
-        .add("settings.global.read.s1");
+        .add("settings.global.read.s1.t1");
     assertThat(SettingsStorage.getCqlLimitPermissions(perms, null),
-        contains("(scope = \"s1\" and userId <> \"\")"));
+        contains("(scope == \"s1.t1\" and userId <> \"\")"));
   }
 
   @Test
   public void getLimitsFromUsers() {
     JsonArray perms = new JsonArray()
         .add("settings.users.read.s1");
     assertThat(SettingsStorage.getCqlLimitPermissions(perms, null),
-        contains("(scope = \"s1\" and userId = \"\")"));
+        contains("(scope == \"s1\" and userId = \"\")"));
```

Comment:
Maybe `==` for the userId test, too?

Reply:
Read the first commit. Special rules for ""